### PR TITLE
Calls setupMirrorFromCache when fetching through the git-fetcher

### PR DESCRIPTION
**Pull request by arcanis**
Otherwise, the mirror will not be populated with data imported from the cache. Cf #3154

### CODE REVIEW

**Comment by bestander** [comment]
Any test you could come up with?

**Comment by arcanis** [comment]
Do we have tests with git, outside of the resolver?

**Comment by bestander** [comment]
I think there are a couple e2e tests with git, we have a GitHub repo for
basic e2e tests.

If setting one up is too much a hassle we could merge it without it but
better with it

File: __tests__/commands/install/integration.js [hunk 1]
```diff
@@ -61,6 +61,24 @@ test.concurrent('properly find and save build artifacts', async () => {
...
+  await runInstall({}, 'install-git', async (config, reporter): Promise<void> => {
+    console.log('aaaaa');
+    const lockfile = await Lockfile.fromDirectory(config.cwd);
+
+    expect(await fs.glob('example-yarn-package.git-*', {cwd: `${config.cwd}/offline-mirror`})).toHaveLength(1);
+
+    await fs.unlink(path.join(config.cwd, 'offline-mirror'));
+    await fs.unlink(path.join(config.cwd, 'node_modules'));
+
+    console.log('bbbbb');
```
Comment (bestander) [inline __tests__/commands/install/integration.js]: nice console.log! :)

**Pull request merged**